src/components/technologies/genericTechnologies/conv3.py

```diff
@@ -169,6 +182,8 @@ def construct_tech_model(self, b_tec, data: dict, set_t_full, set_t_clustered):
         dynamics = self.processed_coeff.dynamics
         rated_power = self.input_parameters.rated_power
 
+        self.big_m_transformation_required = 1
```

Comment:
not sure if we need it here as well since you can only use standby power with perf 2 or 3, which has a big m trans anyway

Reply:
I removed it from the others, as conv3 always needs it

Reply:
perfect

Reply:
Ahh shit, you are right... I changed it.